tests/test_scsv.py

```diff
@@ -98,7 +87,7 @@ def test_read_specfile():
 
 
 @pytest.mark.skipif(_utils.in_ci("win32"), reason="Items are not equal")
```

Comment:
Should we try to see if this test now passes on Windows?

Reply:
I'll check locally on the laptop and remove that skip if it works.

Reply:
It still fails for an unrelated reason. There is an extra `\n` element in one but not the other file when doing `readlines()` on Windows for some reason. Will swap to `read().splitlines()` and filtering out empty list elements `''`.